.github/ISSUE_TEMPLATE/broken-site-issue.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Broken site issue
+about: Report a site where autoconsent is not working
+title: 'Site breakage: [SITE]'
+labels: broken site
+assignees: ''
+
+---
+
+## Description
+
+The popup on [site]
+ - [ ] is not detected
+ - [ ] is not dismissed properly
+ - [ ] other (elaborate below)
+
+**Tested on**
+ - [ ] macOS browser
+ - [ ] Windows browser
+ - [ ] Android
+ - [ ] iOS
+ - [ ] Browser extension
+
+**Tested from region**: US/DE/FR/GB
+
```

Comment:
how about asking for an optional screenshot, like we did in https://app.asana.com/0/1201844467387842/1202975852615920/f?